pkg/rtc/utils.go

```diff
@@ -28,6 +28,25 @@ func PackStreamID(participantID livekit.ParticipantID, trackID livekit.TrackID)
 	return string(participantID) + trackIdSeparator + string(trackID)
 }
 
+func PackSyncStreamID(participantID livekit.ParticipantID, stream string) string {
+	return string(participantID) + trackIdSeparator + stream
+}
+
+func StreamFromTrackSource(source livekit.TrackSource) string {
```

Comment:
Make an assumption here that every publisher will publish only 1 microphone track here, that the stream group for a/v sync will work for 1 audio and N video in a single stream. If there is more than 1 audio in a single stream, the extra audio track will play standalone and might be out of sync.